x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/details_page/components/package_policies/add_integration_flyout.tsx

```diff
@@ -178,7 +165,7 @@ export const AddIntegrationFlyout: React.FunctionComponent<{
               <CreatePackagePolicySinglePage
                 from="policy"
                 queryParamsPolicyId={agentPolicy.id}
-                prerelease={prerelease}
+                prerelease={true}
```

Comment:
We should probably add a similar warning and move to last GA in Add integration flyout

<img width="1528" height="1238" alt="image" src="https://github.com/user-attachments/assets/de7e38fb-22c0-4531-8ce1-542d92e40774" />